src/libsync/networkjobs.cpp

```diff
@@ -294,73 +290,35 @@ bool LsColXMLParser::parse(const QByteArray &xml, QHash<QString, qint64> *sizes,
 
 /*********************************************************************************************/
 
-LsColJob::LsColJob(AccountPtr account, const QUrl &url, const QString &path, QObject *parent)
+PropfindJob::PropfindJob(AccountPtr account, const QUrl &url, const QString &path, int depth, QObject *parent)
     : AbstractNetworkJob(account, url, path, parent)
+    , _depth(depth)
 {
     // Always have a higher priority than the propagator because we use this from the UI
     // and really want this to be done first (no matter what internal scheduling QNAM uses).
     // Also possibly useful for avoiding false timeouts.
     setPriority(QNetworkRequest::HighPriority);
+    Q_ASSERT(depth == 0 || depth == 1);
```

Comment:
This was introduced in 3b3531f77753d1c72909d057e0f79058c0044ae7, but I don't see why it's important. You should probably add a comment.